indico/modules/events/registration/templates/management/_registration_details.html

```diff
@@ -194,6 +196,16 @@
 {% endmacro %}
 
 
+{% macro _render_registration_tags(registration, assigned_tags, all_tags) %}
+    <div class="section" id="registration-detail-registration-tags-assign"
+         data-event-id="{{ registration.registration_form.event.id }}"
+         data-regform-id="{{ registration.registration_form.id }}"
+         data-registration-id="{{ registration.id }}"
+         data-assigned-tags='{{ assigned_tags|tojson }}' data-all-tags='{{ all_tags|tojson }}'>
```

Comment:
I think this may be prone to XSS... Usually we use `="{{ XXX|tojson|forceescape }}"` to ensure things are correctly escaped (tojson by default is meant for script tags, so outside you need forceescape)